adapter/internal/oasparser/utills/open_api_utills_test.go

```diff
@@ -48,16 +49,26 @@ func TestFindSwaggerVersion(t *testing.T) {
 				"basepath": "api/v2"
 							
 				}`,
-			result:  "3",
+			result:  constants.OpenAPIV30,
 			message: "when openAPi version is 3",
 		},
 		{
 			inputSwagger: `{
+				"openapi": "3.1",
 				"host": "petstore.io",
 				"basepath": "api/v2"
 							
 				}`,
-			result:  "2",
+			result:  constants.OpenAPIV31,
+			message: "when openAPi version is 3.1",
```

Comment:
```suggestion
			message: "when openAPI version is 3.1",
```
plz update other places too.

Reply:
Done

Reply:
Updated